### PR TITLE
References cleanup

### PR DESCRIPTION
**Pull request by plehegar**
Makes writing easier

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/performance-timeline/pull/132.html" title="Last updated on Jun 24, 2019, 7:38 PM UTC (1c55d09)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/performance-timeline/132/9fcb9a4...1c55d09.html" title="Last updated on Jun 24, 2019, 7:38 PM UTC (1c55d09)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -538,25 +535,25 @@ <h2><dfn>PerformanceObserverEntryList</dfn> interface</h2>
           <section>
             <h2><dfn>getEntries()</dfn> method</h2>
             <p>Returns a <a>PerformanceEntryList</a> object returned by
-            <a>filter buffer by name and type</a> algorithm with the <a 
-            data-cite="WHATWG-DOM#context-object">context object</a>'s
+            <a>filter buffer by name and type</a> algorithm with the <a
```
Comment (yoavweiss) [inline index.html]: Close the tag on the same line? (or bump the entire tag to the new one?)

File: index.html [hunk 2]
```diff
@@ -538,25 +535,25 @@ <h2><dfn>PerformanceObserverEntryList</dfn> interface</h2>
...
+           >context object</a>'s
             <a>observer buffer</a>, <var>name</var> and <var>type</var> set to
             <code>null</code>.</p>
           </section>
           <section>
             <h2><dfn>getEntriesByType()</dfn> method</h2>
             <p>Returns a <a>PerformanceEntryList</a> object returned by
-            <a>filter buffer by name and type</a> algorithm with the <a 
-            data-cite="WHATWG-DOM#context-object">context object</a>'s
+            <a>filter buffer by name and type</a> algorithm with the <a
```
Comment (yoavweiss) [inline index.html]: Nit: same comment regarding tag end on same line

File: index.html [hunk 3]
```diff
@@ -538,25 +535,25 @@ <h2><dfn>PerformanceObserverEntryList</dfn> interface</h2>
...
             <a>observer buffer</a>, <var>name</var> set to <code>null</code>,
             and <var>type</var> set to the method's input <code>type</code>
             parameter.</p>
           </section>
           <section>
             <h2><dfn>getEntriesByName()</dfn> method</h2>
             <p>Returns a <a>PerformanceEntryList</a> object returned by
-            <a>filter buffer by name and type</a> algorithm with the <a 
-            data-cite="WHATWG-DOM#context-object">context object</a>'s
+            <a>filter buffer by name and type</a> algorithm with the <a
```
Comment (yoavweiss) [inline index.html]: ditto

File: index.html [hunk 4]
```diff
@@ -635,26 +628,23 @@ <h2>Queue a <code>PerformanceEntry</code></h2>
           Call the <a>determine eligibility for adding a performance entry</a>
           algorithm with <var>tuple</var> set to the <a>relevant performance
           entry tuple</a> given <var>entryType</var>.  If it returns true,
-          <a data-cite="WHATWG-INFRA#list-append">append</a> <var>new entry
+          <a  data-xref-for="list">append</a> <var>new entry
```
Comment (yoavweiss) [inline index.html]: Nit: spurious space

**Comment by plehegar** [comment]
@yoavweiss , this should be cleaned now.

**Comment by yoavweiss** [comment]
Still LGTM. Can you rebase?

**Pull request merged**